iron-selectable.html

```diff
@@ -147,7 +147,11 @@
      * @type Array
      */
     get items() {
-      var nodes = Polymer.dom(this).queryDistributedElements(this.selectable || '*');
+      var nodes;
+      if (this.selectable)
+          nodes = this.querySelectorAll(this.selectable);
```

Comment:
I don't think native `querySelectorAll` is how we should find items given that it won't be possible to perform selection into the composed tree when using native shadow DOM.